scripts/windows/RepairFailed64BitMigration.bat

```diff
@@ -0,0 +1,29 @@
+rem This script repairs a broken agent install after encountering the 64-bit OS upgrade bug on v0.10.41 
+@echo off
+rem We don't care about the output of this call, we just use it to verify we are administrator
+net session >nul 2>&1
+if %errorlevel% neq 0 (
+    echo Error: This script must be run as administrator
+    pause

```

Comment:
Do we want these pause commands?  They might get in the way of running this unattended.  I think Grab might be one of those that can run stuff even with the agent down.

Reply:
I can definitely remove them. I added them out of habit to avoid the "prompt pops up and then disappears" thing, but I can definitely see that point